astro_2.0/features/ViewProposal/components/ProposalComments/components/Comment/Comment.tsx

```diff
@@ -112,6 +118,20 @@ export const Comment: FC<CommentProps> = ({
     }
   }, [commentId, onDelete, showReportModal]);
 
+  function renderReportedMessage() {
+    if (!reportsCount) {
+      return null;
+    }
+
+    return (
+      <div className={styles.reportedInfo}>
+        {isReported
+          ? `${t('comments.youReportedThisMessage')} ${reportsCount}/4`
```

Comment:
Is there sense to move `${reportsCount}/4` to variable and reuse it?

Reply:
good catch! fixed